app/commands/v2/patch_link_set.rb

```diff
@@ -109,17 +109,27 @@ def downstream_live(content_id, locale, orphaned_content_ids)
       end
 
       def validate_schema
-        # There may not be a ContentItem yet.
+        # We allow setting links before the document actually exists.
+        # This means we blindly accept anything regardless of what the schema says.
         return true unless schema_name
 
-        # Do not raise anything yet
-        # Only send errbit notification
-        schema_validator.valid?
+        unless schema_validator.valid?
+          # Do not raise anything yet, only report a warning.
+          # This should be changed to an error once we are confident nothing is sending
+          # broken links.
+          GovukError.notify(
```

Comment:
Bit late to this party, but this is contrary to the approach described in https://docs.publishing.service.gov.uk/manual/errors.html#manually-logged-errors. Would it be better if this just logs to Kibana and/or statsd?

Reply:
Ah. Yeah, this should log to kibana instead. I can raise another PR but probably won't get to it until Friday or next week. If anyone wants to raise this now let me know and I'll review.